### DISCUSSION TITLE
Where could I add Usage Examples?

### DISCUSSION THREAD

**Discussion by vardumper** [picocss/pico#283] [Ideas]
Take a look at these examples:
https://codepen.io/vardumper/pen/jOKjjbv
https://codepen.io/vardumper/pen/VwdJoyE

They aren't really components, as they require additional CSS and/or JS.
They also don't fit in the Examples repo, as that repo is more for layout examples.
Nonetheless, I think these sort of examples could be useful to others.

Any opinions? I would like to make a PR, but where should the code go?

**Comment by lucaslarroche** [comment]
Smart examples 🙂 Nice tricks.

I'm not sure where to publish these examples.
I would like the examples and the documentation to be limited to native uses, without additions (the modal component is an exception anyway).

I'm finally working on [v2](https://github.com/picocss/pico/pull/261), and, therefore, on new documentation and a new site. We could add a page for notable and quality uses and examples.

You can also start by creating a repository listing interesting uses and examples.

**Reply by vardumper** [reply]
I added another one: sticky footer. https://codepen.io/vardumper/pen/ZEMzroZ

And I saw someone requested a "Hamburger menu". I think that's also a great candidate for such a usage examples section. I guess this is "frequently needed or requested". I didn't get to push anything to a repo, yet. I'd rather wait until you have re-structured docs, and then join in. (when there's a possible home for such examples).
I did post my 3 examples to my personal site though [link](https://erikpoehler.com/category/playground/)

Cheers
